source/renderer/util/icons.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @ignore
+ * BEGIN HEADER
+ *
+ * Contains:        Clarity icons helper
+ * CVM-Role:        Utility
+ * Maintainer:      Wieke Kanters
+ * License:         GNU GPL v3
+ *
+ * Description:     This module loads the clarity library and adds custom icons.
+ *
+ * END HEADER
+ */
+
+const clarityIcons = require('@clr/icons').ClarityIcons
+require('@clr/icons/shapes/all-shapes')
+const fs = require('fs')
+const path = require('path')
+
+function loadCustomIcons (dir) {
+  fs.readdir(dir, { withFileTypes: true }, (err, list) => {
```

Comment:
I'd recommend using `fs.promises` here, as it makes the text more legible. And, as we don't have to handle the promises generated by asynchronous functions, this should prove to be no problem (we can catch potential errors and log them, to be sure)